### DISCUSSION TITLE
Could you please add a simple example to demonstrate how to use this plugin?

### DISCUSSION THREAD

**Discussion by 722tzz** [zhaoshenzhai/obsidian-mathlinks#66] [Q&A]
I'm a new user of Obsitain, I have read the readme, but I still cannot comprehend how to use it.
So, could you please to add a simple example to demonstrate how to use this plugin?

**Comment by RyotaUshio** [comment]
@zhaoshenzhai The current README was written basically before Obsidian v1.4 comes out, which has introduced [Properties](https://help.obsidian.md/Editing+and+formatting/Properties) (the same thing as YAML front matter/metadata but looks fancy). So we might have to update our documentation to make it easier to understand for new users.

Also, the new properties do not support multi-level YAML: https://forum.obsidian.md/t/properties-support-multi-level-yaml-nested-attributes/63826
which forces users to switch to "source" mode when setting a block mathlink. Maybe we should add a remark on that, too.

(Or it might be helpful to add a new syntax like 
```
mathLink-blocks: ["blockID", "$mathlink goes here$"], ...
```
)

**Comment by LynetteCullens** [comment]
At the moment, My setup is as follows:

1. Use [Glossary Plugin](https://github.com/vschroeter/obsidian-glossary) to maintain auto links to the Title of my equation variables and [https://github.com/RyotaUshio/obsidian-rendered-block-link-suggestions](Render BlockLink Suggestions) so that I can view my links.
2. Use the Metadata Menu Plugin to quickly add frontmatter to all entries located in my Glossary folder.
When creating a new file, it ends up looking like this:
`---
fileClass: Glossary, Glossary
mathLink: 
aliases: 
mathLink-blocks: 
---`
![image](https://github.com/zhaoshenzhai/obsidian-mathlinks/assets/45722611/8c2333ce-04ee-4a06-a96a-378ca49b66d4)

3. Under MathLinks Settings, Remove the "^" under "Edit prefix for block links, and then toggle it off.
4.  Create an entry into the glossary folder Like this:
![image](https://github.com/zhaoshenzhai/obsidian-mathlinks/assets/45722611/4d74a23c-de5f-4f18-b661-77c616062cec)

![image](https://github.com/zhaoshenzhai/obsidian-mathlinks/assets/45722611/2fe87204-4eea-4673-aa88-258fbf63442e)

With mathLink-blocks entries done via the MetaData Menu interface.
![image](https://github.com/zhaoshenzhai/obsidian-mathlinks/assets/45722611/82f0b4fc-19d0-4a1b-b8bb-aa2edc94c3c3)


6. I did all of that and more to create a glossary entry that explains this:
![image](https://github.com/zhaoshenzhai/obsidian-mathlinks/assets/45722611/0821c717-5ec3-422d-b6e3-ff3841488daa)

![image](https://github.com/zhaoshenzhai/obsidian-mathlinks/assets/45722611/a668f451-245f-4217-9f3d-76ec12d660eb)
With Linkage Like this. Now I don't have to remember words, just symbols.
![image](https://github.com/zhaoshenzhai/obsidian-mathlinks/assets/45722611/09a5f15c-78a9-466f-9656-d6229c06ff8c)
